src/test/scala/io/iohk/ethereum/blockchain/sync/SyncControllerSpec.scala

```diff
@@ -332,6 +342,123 @@ class SyncControllerSpec extends FlatSpec with Matchers {
     peer4.expectMsg(PeerActor.Unsubscribe)
   }
 
+  it should "broadcast all blocks if they were all valid" in new TestSetup {
```

Comment:
This test is calling the Ledger in order to validate the block so if by any reason there is a bug within the Ledger, this test fail even if SyncController logic is ok. imo we have two options:

1. Move this test to it
2. Stub ledger execution in this test which might involve in being able to configure Ledger for SyncController

Maybe this is work for another task ( @mcsherrylabs, any thoughts on this? )